### ISSUE TITLE
QR not shown when Require 2FA is enabled

### ISSUE THREAD

**Issue by adriangibanelbtactic** [btactic/zimbra-ose-2fa#1]
Imanudin tried Maldua's zimbra-ose-2fa on Zimbra 8.8.15 and Zimbra 10 OSE unofficial ( https://imanudin.net/2023/09/23/two-factor-authentication-for-zimbra-open-source-edition/ ).

He's using: `./install --compulsory` in its installation so the QR zimlet is being installed.

In his setup he's using: **Require  2FA**. That means that the user is prompted to use 2FA when he logins. So... zimlets are not loaded yet and, of course, no QR is shown in the 2FA wizard.

So, what's needed there is to check if in that scenario zimlets are evenly remotely loaded or not and act accordingly.

After all this might be working well on Zimbra 8.8.15 but not on Zimbra 10 because of other reasons.

**Comment by adriangibanelbtactic** [comment]
Fixed in https://github.com/btactic/zimbra-ose-2fa/tree/v0.7.0 .

**Comment by imanudin11** [comment]
Hello @adriangibanelbtactic . Thanks for the nice add-on. I have tested version 0.7.0 on Zimbra 10 and it works if I disable this line in the TwoFactor_qr.js file

document.querySelectorAll('.ZmTwoFactorSetup')[0].style['height']='446px';

**Comment by adriangibanelbtactic** [comment]
> Hello @adriangibanelbtactic . Thanks for the nice add-on. I have tested version 0.7.0 on Zimbra 10 and it works if I disable this line in the TwoFactor_qr.js file
> 
> document.querySelectorAll('.ZmTwoFactorSetup')[0].style['height']='446px';

Well... you have installed 0.6.0 previously so please make sure to:
```
su - zimbra -c 'zmzimletctl undeploy com_btactic_twofactorauth_qr'
```
before anything else.

Then please create a new issue and report your problem there so that we can take a look at it.

This specific bug has been fixed for 8.8.15 and should not be reopen.
